sled-agent/src/bootstrap/agent.rs

```diff
@@ -151,8 +167,16 @@ impl Agent {
         if subnet_path.exists() {
             info!(agent.log, "Sled already configured, loading sled agent");
             let sled_request: SledAgentRequest = toml::from_str(
-                &tokio::fs::read_to_string(&subnet_path).await?,
-            )?;
+                &tokio::fs::read_to_string(&subnet_path).await.map_err(
+                    |err| BootstrapError::Io {
+                        message: format!(
+                            "Reading subnet path from {subnet_path:?}"
```

Comment:
I guess I could pass it out-of-band, but do you know how to call the `display()` method on implicitly named arguments? I cannot call `format!("Reading subnet path from {subnet_path.display()}")`

Reply:
I think it would have to be passed out of band; the RFC adding string interpolation [intentionally limited it to single identifiers](https://rust-lang.github.io/rfcs/2795-format-args-implicit-identifiers.html#alternative-solution---interpolation).